graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/ChangeUserRequest.java

```diff
@@ -30,6 +32,7 @@
     @Constraints.Required
     public String email;
     public List<String> permissions = Lists.newArrayList();
+    public Set<String> roles = Sets.newHashSet();
```

Comment:
Could `roles` (and `permissions`) be initialized with `Collections.emptySet()` (or `Collections.emptyList()`)?